plutus-tx-plugin/test/Budget/WithoutGHCOptimisations.hs

```diff
@@ -0,0 +1,11 @@
+{-# OPTIONS_GHC
+ -O0 -fmax-simplifier-iterations=0
+ -fno-omit-interface-pragmas
+ -fno-ignore-interface-pragmas
+#-}
```

Comment:
nit for the future, we generally avoid multi-line pragmas, it's a bit less diff-friendly than doing one pragma per line with one option each